downloader.py

```diff
@@ -115,6 +115,7 @@ def grb2_to_array(filename):
         assert(v[i]['level'] == level)
         dataset[0][i] = u[i].data()[0][::2, ::2] # Takes the second element of every second array in data
         dataset[1][i] = v[i].data()[0][::2, ::2]
+    dataset = np.insert(dataset, 360, 0, axis=3)
```

Comment:
Oh I see what you mean. This won't do the right thing, though --- you want to insert the 0th column, not the literal value 0. Also, isn't longitude `axis=1` anyways?